src/components/NavBar/NavMenu.jsx

```diff
@@ -52,32 +52,33 @@ const NavMenu = ({ menuId, openMenu, setOpenMenu, anchorEl, setAnchorEl, setShow
             Notifications
           </Button>
         </MenuItem>
-        <MenuItem>
-          <Button
-            variant="text"
-            startIcon={<AccountCircle />}
-            href={session.info.webId}
-            target="_blank"
-            rel="noreferrer"
-          >
-            Profile
-          </Button>
+        <MenuItem
+          component={Button}
+          variant="text"
+          startIcon={<AccountCircle />}
+          href={session.info.webId}
+          target="_blank"
+          rel="noreferrer"
+          sx={{ color: 'rgba(1, 121, 105, 1)' }}
```

Comment:
Think we could use the green color from our existing theme.js

Reply:
From my (limited) research, it seems that MenuItems do not use the theme palette by default. I have updated theme.js to include this component with the two variants (green/primary and red/error).

Reply:
Think we could have them use the palettes via the `sx` attribute, see clip.

https://github.com/codeforpdx/PASS/assets/14917816/27db94c1-49db-411d-9897-4ae3bc4731cf

Just need to swap what you've made before with the names for the specific color in our palette.

Reply:
⬆️ This is the solution so we can just change the colors in our theme whenever, and it'll be consistent app wide.

Reply:
The green is `primary`, the dark red is `secondary`, and that gold/mustard yellow is `tertiary`.   I'm not against leaving logout the `status` color though. Either red works for me.